### PR TITLE
chore: make D1's experimental backend the default

### LINKED ISSUE #3667: 🚀 Feature Request: Default D1 to the experimental-backend, rather than the WASM backend
### Describe the solution

.

### PR DESCRIPTION
**Pull request by rozenmd**
Fixes #3667 

**What this PR solves / how to test:**

`wrangler d1 create <dbname>` 
`wrangler d1 info <dbname>`

**Associated docs issue(s)/PR(s):**

- N/A

**Author has included the following, where applicable:**

- ~[ ] Tests~
- [x] Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))

**Reviewer is to perform the following, as applicable:**

- Checked for inclusion of relevant tests
- Checked for inclusion of a relevant changeset
- Checked for creation of associated docs updates
- Manually pulled down the changes and spot-tested

### CODE REVIEW

File: packages/wrangler/src/d1/create.tsx [hunk 1]
```diff
@@ -59,8 +54,8 @@ export const Handler = withConfig<HandlerOptions>(
 				},
 				body: JSON.stringify({
 					name,
+					experimental: true,
```
Comment (rozenmd) [inline packages/wrangler/src/d1/create.tsx:57]: We can remove this once D1's API defaults to the new backend.

**Review by elithrar — APPROVED** [review]
LGTM. Let's go!

File: packages/wrangler/src/d1/create.tsx [hunk 2]
```diff
@@ -81,13 +76,11 @@ export const Handler = withConfig<HandlerOptions>(
...
-						<Text>
-							Created your database using D1&apos;s new storage backend. The new
-							storage backend is not yet recommended for production workloads
-							and does not yet support backups.
-						</Text>
-					)}
+					<Text>
+						Created your database using D1&apos;s new storage backend. The new
+						storage backend is not yet recommended for production workloads, but
+						backs up your data via point-in-time restore.
```
Comment (Skye-31) [inline packages/wrangler/src/d1/create.tsx:82]: Should this message be updated?

Comment (rozenmd) [reply packages/wrangler/src/d1/create.tsx:82]: Eh we can update that once we go from alpha -> beta

**Pull request merged**